services/core-web/src/components/Forms/noticeOfWork/UpdateNOWLeadInspectorForm.js

```diff
@@ -1,34 +1,70 @@
 import React from "react";
 import { Field, reduxForm } from "redux-form";
+import { Button, Popconfirm } from "antd";
 import { Form } from "@ant-design/compatible";
 import "@ant-design/compatible/assets/index.css";
 import PropTypes from "prop-types";
 import { required } from "@common/utils/Validate";
 import * as FORM from "@/constants/forms";
 import { renderConfig } from "@/components/common/config";
 import CustomPropTypes from "@/customPropTypes";
+import AuthorizationWrapper from "@/components/common/wrappers/AuthorizationWrapper";
+import * as Permission from "@/constants/permissions";
 
 const propTypes = {
   inspectors: CustomPropTypes.groupOptions.isRequired,
   setLeadInspectorPartyGuid: PropTypes.func.isRequired,
+  handleSubmit: PropTypes.func.isRequired,
+  isAdminView: PropTypes.bool.isRequired,
+  isEditMode: PropTypes.bool.isRequired,
+  setEditMode: PropTypes.func.isRequired,
+  title: PropTypes.bool.isRequired,
 };
 
 const UpdateNOWLeadInspectorForm = (props) => {
   return (
     <div>
-      <Form layout="vertical">
+      <Form layout="vertical" onSubmit={props.handleSubmit}>
+        <div className="field-title">Lead Inspector</div>
         <Form.Item>
           <Field
             id="lead_inspector_party_guid"
             name="lead_inspector_party_guid"
-            label="Lead Inspector"
+            label={
+              !props.isAdminView
+                ? "Assign a lead inspector before continuing. This assignment can be updated later under the Administrative tab."
+                : ""
+            }
             component={renderConfig.GROUPED_SELECT}
-            placeholder="Start typing lead inspector name"
+            placeholder="Start typing lead inspectors name"
```

Comment:
Should be "lead inspector's name", but this still sounds weird without an article like "a" or "the"...
What about "Enter Lead Inspector Name" or "Start typing the lead inspector's name"?